dpdata.py

```diff
@@ -159,6 +160,12 @@ def __fixup_nomail_flag(self):
                 # Empty out NOMAIL_REASON if NOMAIL not set
                 dp_donorrecord['NOMAIL_REASON'] = ''
 
+    def __copy_spouse_email(self):
```

Comment:
I got an email. The reasons I put this in the cleanup code:
- Had existing data to clean up
- No dependence on district data
- Otherwise would have had to put it in 2 places: new donor and update existing donor. The update existing donor logic is a fair bit more complicated

I think you're right, this was working before in the new donor code. I think too that in one of the cases we were setting SPOUSE_EMAIL in the case where there is no Parent1 but only Parent2.